documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -48,22 +39,61 @@ $ git clone --depth=1 --branch <branch> https://github.com/raspberrypi/linux
 For a full list of available branches, see the https://github.com/raspberrypi/linux[the Raspberry Pi kernel repository].
 ====
 
-==== Apply configuration
+Now that you have the kernel source, build a fresh kernel xref:linux_kernel.adoc#natively-build-a-kernel[natively] or via xref:linux_kernel.adoc#cross-compiled-build[cross-compilation].
+
+=== Natively build a kernel
+
+On a Raspberry Pi, install the latest version of xref:os.adoc[Raspberry Pi OS]. Boot your Raspberry Pi, log in, and connect to the Internet.
```

Comment:
Not sure if the second sentence here is necessary - isn't it kinda obvious? Also, presumably you've already had to do each of those three tasks to complete the previous "Download kernel source" step?